projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -75,29 +75,30 @@ const BoostSpeedScore: FC = () => {
 		}
 
 		setIsLoading( true );
-		try {
```

Comment:
FYI @elliottprogrammer when you come back, just wanted to let you know I restructured this try/catch async/await to a `.then()` function because the try catch block was failing and producing some unspecific and quiet errors if the actual setting of the scores was failing

I added destructuring to the functions above and it was failing due to a null object, but that wasn't being logged to the console because the catch block was catching it and not logging it anywhere I could see. This should make errors like that more visible in the future 😄

Reply:
Excellent. Sounds good @CodeyGuyDylan! Thanks for locating and applying these improvements! 👍😃